data_diff/databases/base.py

```diff
@@ -202,6 +202,16 @@ class BaseDialect(abc.ABC):
     TYPE_CLASSES: ClassVar[Dict[str, Type[ColType]]] = {}
 
     PLACEHOLDER_TABLE = None  # Used for Oracle
+    USE_TOP_INSTEAD_LIMIT: bool = False  # True for MsSQL or Teradata
+
+    # Some database do not support long string so concatenation might lead to type overflow
+    PREVENT_OVERFLOW_WHEN_CONCAT: bool = False
+
+    _prevent_overflow_when_concat: bool = False
```

Comment:
did you mean to have two of these variables?

Reply:
Yep, i though to introduce this param as internal but it is still classvar so it is not a good option. I will rewrite it.